doc/default.nix

```
@@ -46,30 +46,47 @@ let
 in pkgs.stdenv.mkDerivation {
   name = "nixpkgs-manual";
 
-  nativeBuildInputs = with pkgs; [
-    pandoc
-    graphviz
-    libxml2
-    libxslt
-    zip
-    jing
-    xmlformat
-  ];
+  nativeBuildInputs = with pkgs; [ nixos-render-docs ];
```

Keep the diff clean for now and future additions?

```suggestion
  nativeBuildInputs = with pkgs; [
    nixos-render-docs
  ];
```